src/node/coinstats.cpp

```diff
@@ -75,9 +77,33 @@ bool GetUTXOStats(CCoinsView* view, CCoinsStats& stats, const std::function<void
         pcursor->Next();
     }
     if (!outputs.empty()) {
-        ApplyStats(stats, ss, prevkey, outputs);
+        ApplyStats(stats, hash_obj, prevkey, outputs);
     }
-    stats.hashSerialized = ss.GetHash();
+
+    FinalizeHash(hash_obj, stats);
+
     stats.nDiskSize = view->EstimateSize();
     return true;
 }
+
+bool GetUTXOStats(CCoinsView *view, CCoinsStats &stats, CoinStatsHashType hash_type, const std::function<void()>& interruption_point)
+{
+    switch(hash_type) {
+        case(CoinStatsHashType::HASH_SERIALIZED): {
+            CHashWriter ss(SER_GETHASH, PROTOCOL_VERSION);
+            return GetUTXOStats(view, stats, ss, interruption_point);
+        }
+    }
+    return false; // Unreachable code
```

Comment:
87dd07e0382c6398d118d88c88b0c9ef24251e37:

According to the developer notes, assert(false) and with the appropriate comment

Reply:
done